packages/api/src/controllers/playback.ts

```diff
@@ -66,7 +71,13 @@ function newPlaybackInfo(
     type: "html5/application/vnd.apple.mpegurl",
     url: hlsUrl,
   });
-
+  if (webRtcUrl) {
+    playbackInfo.meta.source.push({
+      hrn: "WebRTC (VP8)",
+      type: "html5/video/vp8",
```

Comment:
Does this mean that Mist will be transcoding everything to VP8? 😮

Reply:
I am not sure what the `hrn` should be here. What would you include instead?

Reply:
Oh yeah I had just made that same comment in the other file. I don't know what else we could set for the `hrn`, seems like we're restricted to MIME types.

The confusion about VP8 is probably something else tho, assuming that you got these values like `WebRTC (VP8)` from Mist. Curious about how that video ends up in VP8 codec if it goes in/out of go-livepeer as H264/5